go.mod

```diff
@@ -2,7 +2,9 @@ module github.com/ipfs/go-ipfs-blockstore
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gammazero/kmutex v1.0.0
```

Comment:
This needs to be updated to `v1.0.1`.  The `v1.0.0` tag was pulling in the old version that this was forked from, which is not what you want.

Reply:
The maintainer of the original repo accepted my changes, so now you can just use:
```
github.com/im7mortal/kmutex v1.0.1
```